cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -816,3 +816,14 @@ func (g *genConversion) doAlias(inType, outType *types.Type, sw *generator.Snipp
 func (g *genConversion) doUnknown(inType, outType *types.Type, sw *generator.SnippetWriter) {
 	sw.Do("// FIXME: Type $.|raw$ is unsupported.\n", inType)
 }
+
+func isDirectlyAssignable(inType, outType *types.Type) bool {
```

Comment:
It seems that in all places you are using the arguments the other way (output, input) and the signature is different.

I would prefer leaving the signature as is and changing calls...